pkg/cmd/render/render.go

```diff
@@ -286,6 +294,14 @@ func newTemplateData(opts *renderOpts) (*TemplateData, error) {
 		return nil, err
 	}
 
+	// Use a marker file to drive configure the bootstrap scaling strategy.
+	if _, err := os.Stat(opts.nonHABootstrapScalingStrategyMarker); err == nil {
```

Comment:
do u want to log?

Reply:
I went ahead and added logging for when the file _does exist_, because the vast majority of the time it won't. If we care enough, we could do a follow-up to the installer to perform the existence check and pass the marker file argument from the call side (bootkube.sh), and then I could remove the defaulting here and actually make render return an error if passed a file that doesn't exist.